pennylane/pulse/rydberg.py

```diff
@@ -40,7 +40,7 @@ def rydberg_interaction(
         V_{ij} = \frac{C_6}{R_{ij}^6}
 
     where :math:`R_{ij}` is the distance between the atoms :math:`i` and :math:`j`, and :math:`C_6`
-    is the Rydberg interaction constant, which defaults to :math:`862690 \text{MHz} \times \mu \text{m}^6`.
+    is the Rydberg interaction constant, which defaults to :math:`5420000 \text{MHz} \times \mu \text{m}^6`.
```

Comment:
This seems consistent, yes. So then the braket notebook indeed would be talking about Hz rather than rad/s.

Reply:
No, the unit in the braket notebook is in rad/s as stated. The unit in PL is currently in MHz, and possibly should stay that way, pending resolution of units conversation at 11.

5.42e-24 rad/s = 5.42/(2pi)e-24 Hz 
                           = 0.86261979155e-24 Hz
                           = 862620e-30 Hz
                           = 862620e-36 MHz
                           
Then the factor e-36 is cancelled out by the fact that we are using $\mu m^6$ instead of $m^6$.

Another way of saying this is that when you multiply 862690 by 2pi, what you get out is an angular frequency, i.e. its in units of rad/s, not Hz. I.e. if $\omega = 2 \pi f$, and $f$ has units Hz, then $\frac{\omega}{2 \pi}$ has units Hz, and $2 \pi f$ has units rad/s.

Reply:
> we have $\text{rad}/\text{s} = 2 \pi \text{Hz}$ no?

Other way around. $\text{Hz} = 2 \pi \text{rad}/\text{s}$

Reply:
> $\frac{\omega}{2 \pi}$ has units Hz

Does that not conclude that $\frac{\text{rad}}{2 \pi s} = \text{Hz}$ 🤔

Reply:
I think maybe a better way to think about it is to say 1Hz is one full cycle per second, and if one full cycle is defined as going around a unit circle, it's $2 \pi$ radians. So 1Hz = one cycle per second = $2 \pi$ rad/s. 1 --> $2 \pi$.

Don't think about it algebraically. Think about it as a shift in your point of reference, from once = 1 to once = $2 \pi$.

Its confusing because when papers write things like $\omega = 2 \pi \times (5 GHz)$, it looks like if you multiply that together you will have around 30 GHz. But you won't - multiplying by $2 \pi$ means you shifted your point of reference, so now you are in rad/s. $2 \pi \times (5 \text{GHz}) = 2 \pi \times 5e9 \text{Hz} = 31.4e9 \text{ rad/s}$. Multiplying by $2 \pi$ changes the number _and_ the unit, so you can't treat it like a normal algebraic thing where you are "moving" a factor of $2 \pi$ between the unit and the number.

Reply:
Thanks for that!

It's really odd that it algebraically does not make sense. Like from that calculation you are doing we have $2 \pi \times 5 \times 10^6 \text{Hz} = 31.4 \times 10^6 \text{rad}/s$, which implies $\text{Hz} = \text{rad}/s$ algebraically. So it really is more a matter of "do I write out the explicit $2 \pi$ factor, or not?" - for which $\text{Hz}$ is the answer to the former and $\text{rad}$ the answer to the latter. This is more confusing than it probably should be 😕 
> Multiplying by changes the number and the unit

🤯 That seems to be the crux of the story 😅